docs/specifications/cva6_requirement_specification.adoc

```diff
@@ -0,0 +1,391 @@
+= CVA6 requirement specification
+
+Revision 0
+
+== License
+
+Copyright 2022 OpenHW Group and Thales +
+Copyright 2018 ETH Zürich and University of Bologna +
+SPDX-License-Identifier: Apache-2.0 WITH SHL-2.1 +
+Licensed under the Solderpad Hardware License v 2.1 (the “License”); you may not use this file except in compliance with the License, or, at your option, the Apache License version 2.0. You may obtain a copy of the License at https://solderpad.org/licenses/SHL-2.1/[https://solderpad.org/licenses/SHL-2.1/].  +
+Unless required by applicable law or agreed to in writing, any work distributed under the License is distributed on an “AS IS” BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+
+== Introduction
+
+CVA6 is a RISC-V compatible application processor core that can be configured as a 32- or 64-bit core (RV32 or RV64). It includes L1 caches, optional MMU, optional PMP and optional FPU.
+
+It is an industrial evolution of ARIANE created by ETH Zürich and the University of Bologna. It is written in SystemVerilog and maintained by the OpenHW Group.
+
+This specification is organized as requirements that apply to the “Scope of the IP”.
+
+The requirement list is to be approved by the OpenHW Group Technical Work Group (TWG), as well as its change requests.
+
+The specification will be complemented by a user’s guide.
+
+Revision 0 refers to the product of the first CVA6 project led at OpenHW Group. It is a placeholder in case of future evolutions after project freeze (PF gate).
+
+A list of abbreviations is available at the end of this document.
```

Comment:
This is simply a test of adding a comment to this review.  Please ignore.

Reply:
Thanks for this comment which is easy to resolve.